RealmSwift/Aliases.swift

```diff
@@ -17,7 +17,7 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import Foundation
-import Realm
+@_exported import Realm
```

Comment:
typealiasing something from an imported module will require users of the typealias to import that module in Swift 6. This change preserves the old behavior.